index.js

```diff
@@ -27,13 +27,20 @@ const SCHEMA_SLACK_SETTINGS = Joi.object().keys({
         SCHEMA_SLACK_CHANNELS, SCHEMA_SLACK_CHANNEL
     )
 }).unknown(true);
+const SCHEMA_PIPELINE_DATA = Joi.object()
+    .keys({
+        scmRepo: Joi.object().keys({ name: Joi.string() }).required()
+    }).unknown(true);
 const SCHEMA_BUILD_DATA = Joi.object()
     .keys({
         settings: SCHEMA_SLACK_SETTINGS.required(),
         status: SCHEMA_STATUS.required(),
-        pipelineName: Joi.string(),
+        pipeline: SCHEMA_PIPELINE_DATA.required(),
         jobName: Joi.string(),
-        buildId: Joi.number().integer(),
+        build: Joi.object().keys({
+            id: Joi.number().integer().required()
+        }).unknown(true),
+        event: Joi.object().required(),
```

Comment:
don't think `event` object is required for our notifications (at the moment)

Reply:
Remove required from `event` object.